src/components/SRNEntry.vue

```diff
@@ -40,10 +40,10 @@
 
       <div class="my-auto px-3" v-show="ifUserEnteredMoreThanOne">
         <button
-          class="material-icons text-red-500"
           @click="removeField(index, userIDList)"
         >
-          remove_circle
+          <inline-svg
+          :src="require('@/assets/images/remove_circle.svg')"></inline-svg>
```

Comment:
(read this comment after reading the comment attached to add_circle.svg below) 
You'll have to do the same thing here, as discussed in the other comment, remove all the fill, height, width tags from the original svg file and then specify height and width here